filters/filtertest/filtertest.go

```diff
@@ -60,6 +63,15 @@ func (fc *Context) ParentSpan() opentracing.Span {
 	return opentracing.StartSpan("test_span")
 }
 
+func (fc *Context) Logger() filters.FilterContextLogger {
+	traceId := tracing.GetTraceID(fc.FRequest)
+	if traceId == "" {
+		return log.WithFields(log.Fields{})
```

Comment:
I think we can `return log.StandardLogger()` here to avoid unnecessary empty fields allocation